### ISSUE TITLE
Show a progress bar while moving Dropbox folder

### ISSUE THREAD

**Issue by raffaem** [SamSchott/maestral#293]
**Describe the bug**
1. Click Maestral icon
2. Preferences
3. Dropbox folder location -> Other
4. Select a new folder
5. Click "Open" on the new folder
6. Maestral hangs

**To Reproduce**
See above

**Expected behaviour**
The files should be copied into the new folder without maestral hanging

**System (please complete the following information):**
 - Maestral version: 1.3.1
 - OS: [e.g. Ubuntu] Ubuntu 20.10
 - Desktop environment: [e.g. Gnome 3.32] GNOME 3.38.1
 - Qt version: Using Qt version 5.14.2 in /usr/lib/x86_64-linux-gnu

**Additional context**

`maestral log show`

```
2021-01-28 20:11:58 maestral.sync DEBUG: Local cursor saved: 1611861118.2470286
2021-01-28 20:12:32 maestral.sync DEBUG: Detected remote changes: False
2021-01-28 20:12:32 maestral.sync DEBUG: Waiting for remote changes since cursor:
<ABRIDGED>
2021-01-28 20:12:38 maestral.sync DEBUG: Local cursor saved: 1611861158.273522
2021-01-28 20:13:18 maestral.sync DEBUG: Local cursor saved: 1611861198.2940478
2021-01-28 20:13:22 maestral.main INFO: Moving Dropbox folder...
2021-01-28 20:13:22 maestral.sync INFO: Shutting down threads...
2021-01-28 20:13:22 maestral.sync INFO: Syncing stopped
2021-01-28 20:13:35 maestral.sync DEBUG: Detected remote changes: False
```

`maestral status`

```
maestral status

Account:      <ABRIDGED>
Usage:        4.7% of 437.6 TB used
Status:       Syncing stopped
Sync threads: Stopped
Sync errors:  0
```

`maestral activity`

Cannot copy output, but says "Syncing stopped`

**Comment by SamSchott** [comment]
"Syncing stopped" is as it should be. But it should also be resumed afterwards. Did the actual folder get moved?

I have an automated integration test for that, so I'm surprised that this should fail.

**Comment by raffaem** [comment]
> "Syncing stopped" is as it should be. But it should also be resumed afterwards. Did the actual folder get moved?
> 
> I have an automated integration test for that, so I'm surprised that this should fail.

Hello,

it actually IS working.

The "problem" is that, when you click "Open" in this screen:

![Untitled](https://user-images.githubusercontent.com/54762742/106263029-1ffdbf00-6224-11eb-9874-4027e892fcd6.png)

Maestral "hangs" on that screen while files are being copied over. That is, you see that screen, and you are unable to close/get rid on it. There is no progress bar, no indication of what is happening. In reality, files are being copied over, and it make that a while.

When it ends to copy to files, that windows closes and maestral goes into "Indexing local changes". I'm not sure what that is / why we need that? But at least it is clear it is not hanging and it is doing something

I think we can change this issue to "show a progress bar while files are being copied over".

**Comment by SamSchott** [comment]
Ahh, good to know! Yes, I'll note this as a feature request. It makes sense to show a progress bar when the move may take some time.

**Comment by raffaem** [comment]
Either a progress bar, and close that window and display "Moving...." on the context menu associated with the systray icon

**Comment by SamSchott** [comment]
> close that window and display "Moving...." on the context menu associated with the systray icon

I think this is what the macOS GUI is doing, but I can quite remember. I'll look into it!

**Comment by SamSchott** [comment]
Closing since this is fixed in 1.4.0. The move will now be carried out in the background, similar to the macOS GUI.
